blocks/block-alignment-toolbar/index.js

```diff
@@ -25,6 +25,10 @@ const BLOCK_ALIGNMENTS_CONTROLS = {
 		icon: 'align-full-width',
 		title: __( 'Full width' ),
 	},
+	fixed: {
```

Comment:
Looking at the code for `TextboxControl` it looks like I can pass `type="checkbox"` - how do you feel about me renaming that to `InputControl`?